How-to-contribute.md

```diff
@@ -1,22 +1,20 @@
-<h1> HOW TO CONTRIBUTE </h1>
+<h1> How To Contribute & Earn XUC </h1>
 
-Thank you for considering to help in building Exchange Union! We welcome any contributions no matter how big or small - from typos to bug fixes, you will be awarded with XUC (Exchange Union's token) after carefully analyzing every pull request. Github is our git version control service of choice, used to track issues, suggestions, feature request, documentations and is home to the source code which powers Exchange Union. 
+Yay! Kudos for being a part of Exchange Union & help building XUD, our node software! Any contributions are welcome no matter how big or small. From typos and bug fixes to entire modules - you will be awarded with XUC, our token.
```

Comment:
"helping build" instead of "help building."

Reply:
Fixed